tr/news/_posts/2019-10-22-ruby-2-7-0-preview2-released.md

```diff
@@ -0,0 +1,262 @@
+  [[Özellik #15901]](https://bugs.ruby-lang.org/issues/15901)
+
+  ```ruby
+  a = %w(foo bar baz)
+  e = a.lazy.map {|x| x.upcase }.map {|x| x + "!" }.eager
+  p e.class               #=> Enumerator
+  p e.map {|x| x + "?" }  #=> ["FOO!?", "BAR!?", "BAZ!?"]
+  ```
+
+## Performans iyileştirmeleri
+
+* JIT [Deneysel]
+
+  * JIT edilmiş kod, bir eniyileştirme varsayımı geçersiz kılındığında, daha az eniyileştirilmiş koda yeniden derlenir.
+
+  * Bir metodun saf olduğu düşünülürse, metod satıriçi yapma uygulanır.
+    Bu eniyileme hala deneyseldir ve birçok metod henüz saf olarak düşünülmemektedir.
+
+  * `--jit-min-calls`'ın varsayılan değeri 5'ten 10.000'e değiştirildi.
+
+  * `--jit-max-cache`'in varsayılan değeri 1.000'den 100'e değiştirildi.
+
+* `Symbol#to_s`, `Module#name`, `true.to_s`, `false.to_s` ve `nil.to_s` artık her zaman donmuş String döndürüyor.
+Döndürülen String, belirli bir nesne için her zaman aynıdır.
+[Deneysel]  [[Özellik #16150]](https://bugs.ruby-lang.org/issues/16150)
+
+* `CGI.escapeHTML`'in performansı iyileştirildi.
+[GH-2226](https://github.com/ruby/ruby/pull/2226)
+
+## 2.6'dan bu yana dikkate değer diğer değişiklikler
+
+* Bazı standart kütüphaneler güncellendi.
+  * Bundler 2.1.0.pre.1
+  * RubyGems 3.1.0.pre.1
+  * CSV 3.1.2 ([NEWS](https://github.com/ruby/csv/blob/v3.1.2/NEWS.md))
+  * Racc 1.4.15
+  * REXML 3.2.3 ([NEWS](https://github.com/ruby/rexml/blob/v3.2.3/NEWS.md))
+  * RSS 0.2.8 ([NEWS](https://github.com/ruby/rss/blob/v0.2.8/NEWS.md))
+  * StringScanner 1.0.3
+  * Orijinal sürümü olmayan bazı diğer kütüphaneler de güncellendi.
+
+* Bir blokla çağrılmış bir metodda bloksuz olarak yer alan `Proc.new` ve `proc` şimdi bir uyarıdır.
+
+* Bir blokla çağrılmış bir metodda bloksuz olarak yer alan `lambda` şimdi bir hatadır.
+
+* Unicode ve Emoji sürümleri 11.0.0'dan 12.0.0'a güncellendi.
+  [[Özellik #15321]](https://bugs.ruby-lang.org/issues/15321)
+
+* Unicode sürümü 12.1.0'a güncellendi, bu sürüm U+32FF SQUARE ERA NAME REIWA için destek eklemektedir.
+  [[Özellik #15195]](https://bugs.ruby-lang.org/issues/15195)
+
+* `Date.jisx0301`, `Date#jisx0301`, ve `Date.parse` yeni Japon dönemini desteklemektedir.
+  [[Özellik #15742]](https://bugs.ruby-lang.org/issues/15742)
+
+* Derleyicilerin C99'u desteklemesi zorunlu tutuldu.
+  [[Diğer #15347]](https://bugs.ruby-lang.org/issues/15347)
+  * Lehçemizin ayrıntıları: <https://bugs.ruby-lang.org/projects/ruby-trunk/wiki/C99>
+
+Daha fazla ayrıntı için [NEWS](https://github.com/ruby/ruby/blob/v2_7_0_preview2/NEWS)'e ya da [işleme logları](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview2)na bakın.
+
+Bu değişiklikler ile birlikte Ruby 2.6.0'dan bu yana [3670 dosya değişti, 201242 ekleme(+), 88066 silme(-) yapıldı](https://github.com/ruby/ruby/compare/v2_6_0...v2_7_0_preview2)!
+
+Ruby 2.7 ile programlamanın tadını çıkarın!
+
+## İndirin
+
+* <https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.0-preview2.tar.bz2>
+
+      BOYUT:  14555229 bayt
```

Comment:
@ismailarilik This section is never translated, but kept as in the `en` version.